### PR TITLE
Refine Liqoctl Offloading

### PR DESCRIPTION
**Pull request by palexster**
# Description

This PR refactors the `liqoctl offload command` by adding a message in case of success.

# How Has This Been Tested?

- [x] Unit Tests

### CODE REVIEW

File: pkg/liqoctl/install/helm.go [hunk 1]
```diff
@@ -61,8 +61,8 @@ func initLiqoRepo(helmClient helm.Client) error {
 		return err
 	}
 
-	// This repository is required to install the liqo clastix dependency.
-	// The same values are inserted in Chart.yaml file and should be updated accordingly.
+	// This repository are required to install the liqo clastix dependency
```
Comment (aleoli) [inline pkg/liqoctl/install/helm.go:64]:
```suggestion
	// This repository is required to install the liqo clastix dependency
```
or
```
These repositories are required
```

File: cmd/liqoctl/cmd/offload.go [hunk 2]
```diff
@@ -25,29 +25,38 @@ import (
...
 	}
 	podOffloadingStrategy := args.NewEnum([]string{string(offloadingv1alpha1.LocalAndRemotePodOffloadingStrategyType),
 		string(offloadingv1alpha1.RemotePodOffloadingStrategyType),
-		string(offloadingv1alpha1.LocalPodOffloadingStrategyType)},
-		string(offloadingv1alpha1.LocalAndRemotePodOffloadingStrategyType))
+		string(offloadingv1alpha1.LocalPodOffloadingStrategyType)}, string(offloadingv1alpha1.LocalAndRemotePodOffloadingStrategyType))
 
-	offloadClusterCmd.PersistentFlags().Var(podOffloadingStrategy,
-		offload.PodOffloadingStrategyFlag, offload.PodOffloadingStrategyHelp)
+	offloadClusterCmd.PersistentFlags().Var(podOffloadingStrategy, offload.PodOffloadingStrategyFlag, "Select the desired pod offloading strategy")
```
Comment (aleoli) [inline cmd/liqoctl/cmd/offload.go:53]: Are these changes desired? Should we explain which are the supported values?

**Review by aleoli — APPROVED** [review]

File: cmd/liqoctl/cmd/offload.go (hunk 2 above)
Comment (palexster) [reply cmd/liqoctl/cmd/offload.go:53]: Thanks for noticing.

File: pkg/liqoctl/offload/consts.go [hunk 3]
```diff
@@ -33,16 +33,21 @@ $ liqoctl offload namespace liqo-demo --namespace-mapping-strategy=EnforceSameNa
 	ClusterResourceName = "namespace"
 	// SuccessfulMessage is printed when a Offload cluster command has scucceded.
 	SuccessfulMessage = `
-	Success 👌!
+	Success 👌! The offloading rules for the namespace %s has been created on the cluster! 🚀
+	Check them out by typing:
```
Comment (giorio94) [inline pkg/liqoctl/offload/consts.go:37]:
```suggestion
	Success 👌! The offloading rules for namespace %s have been created in your cluster! 🚀
	Check them out typing: 
```

File: pkg/liqoctl/install/helm.go [hunk 4]
```diff
@@ -61,8 +61,8 @@ func initLiqoRepo(helmClient helm.Client) error {
 		return err
 	}
 
-	// This repository is required to install the liqo clastix dependency.
-	// The same values are inserted in Chart.yaml file and should be updated accordingly.
+	// This repository is required to install the liqo clastix dependency
+	// The same values are available in Chart.yaml file.
```
Comment (giorio94) [inline pkg/liqoctl/install/helm.go:65]: Is this change intentional?

File: pkg/liqoctl/generate/consts.go [hunk 5]
```diff
@@ -18,7 +18,10 @@ package generate
 const LiqoctlGenerateShortHelp = "Generate the command to execute on another cluster to peer with the selected cluster."
 
 // LiqoctlGenerateLongHelp contains the short help message for the generate-add-cluster Liqoctl command.
-const LiqoctlGenerateLongHelp = "Generate the command to execute on another cluster to peer with the selected cluster."
+const LiqoctlGenerateLongHelp = `Generate the command to execute on another cluster to peer with the selected cluster.
+
+liqoctl generate-add-command command requires Liqo to be installed on the target cluster.
```
Comment (giorio94) [inline pkg/liqoctl/generate/consts.go:23]: To me, this sentence looks misleading, as I feel you are referring to the cluster you want to peer with (target), and not the one you are executing this command on.

**Review by giorio94 — APPROVED** [review]
